### ISSUE TITLE
Add Exception Stacks view when viewing a .nettrace with exceptions

### ISSUE THREAD

**Issue by mconnew** [microsoft/perfview#1797]
When collecting a trace using `dotnet-trace collect`, the generated .nettrace file contains call stacks for any exceptions thrown. When using the Events view, I can filter the events to `Microsoft-Windows-DotNETRuntime/Exception/Start` and see that it contains the exception type and exception message. When using the Any Stacks view, I can find the stack using the event time as the Start and End time on the view and see the full call stack associated with the exception. Currently there's no way to view what exception the call stack is for from a stack based view of events.  

I believe all the information needed is already there to provide the same Exception Stacks view as is available when using an .etl collected trace.

**Comment by brianrob** [comment]
Yes, you're right - this is just a matter of porting the view.  Should be pretty straight forward.  I can do this, or if you're interested in submitting a PR, that's always encouraged.  :)
